packages/ui/package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "dev": "yarn build:nexus && next dev",
     "build": "yarn build:nexus && next build",
-    "build:nexus": "ts-node-dev -O '{ \"module\": \"commonjs\" }' --transpile-only graphql/schema.ts && prettier --write graphql/generated/*",
+    "build:nexus": "yarn generate-graphql-types && ts-node-dev -O '{ \"module\": \"commonjs\" }' --transpile-only graphql/schema.ts && prettier --write graphql/generated/*",
+    "generate-graphql-types": "graphql-codegen",
```

Comment:
```suggestion
    "build:nexus": "yarn generate-graphql-types && ts-node-dev -O '{ \"module\": \"commonjs\" }' --transpile-only graphql/schema.ts",
    "build:graphql": "yarn build:nexus && yarn build:graphql-codegen && yarn prettier:generated",
    "build:graphql-codegen": "graphql-codegen",
    "prettier:graphql": "prettier --write graphql/generated/*",
```

Reply:
added, prettier is done at the top level.